### PR TITLE
Sdks 2370

### PR DESCRIPTION
**Pull request by rodrigoareis**
# JIRA Ticket

[SDKS-2370](https://bugster.forgerock.org/jira/browse/SDKS-2370) Authenticator Policies - jailbreak and biometric policies

# Description

This changes adds the Authenticator Policies feature into the Authenticator App, enabling blocking accounts with Jailbroken/Rooted devices and enforcing Biometric Authentication. 

# Definition of Done Checklist:

- [x] Acceptance criteria is met.
- [x] All tasks listed in the user story have been completed.
- [x] Coded to standards.
- [x] Ensure backward compatibility.
- [x] API reference docs is updated.
- [x] Unit tests are written.
- [x] Integration tests are written.
- [ ] e2e tests are written.
- [ ] Functional spec is written/updated.
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

### CODE REVIEW

File: forgerock-authenticator/ios/Classes/FRAStorageClient.swift
Comment (spetrov) [inline forgerock-authenticator/ios/Classes/FRAStorageClient.swift:1]: Copyright note?

File: forgerock-authenticator/ios/Classes/FRAppDelegate.swift
Comment (spetrov) [inline forgerock-authenticator/ios/Classes/FRAppDelegate.swift:1]: Update year in copyright notes...

File: forgerock-authenticator/ios/Classes/FRAClientWrapper.swift
Comment (spetrov) [inline forgerock-authenticator/ios/Classes/FRAClientWrapper.swift:1]: Update year in copyright notes...

File: forgerock-authenticator/example/lib/widgets/account_card.dart
Comment (spetrov) [inline forgerock-authenticator/example/lib/widgets/account_card.dart:1]: Update year in copyright notes...

File: forgerock-authenticator/android/libs/forgerock-authenticator-debug.aar
[the remark this answers is not in the document]
Comment (rodrigoareis) [reply forgerock-authenticator/android/libs/forgerock-authenticator-debug.aar:1]: Yes, this should be replaced by the android beta5, when it is released.

**Review by spetrov — APPROVED** [review]
Looks good!

**Pull request merged**